### ISSUE TITLE
Firefox Support

### ISSUE THREAD

**Issue by Shir-e** [IronFlare/Trading-Verifier-Docs#5]
Hello, as an active trader, I have to beware of scammers, like everyone else. But I don't use Chrome (and don't intend to). I use Firefox, due to its superior performance and it being open-source. My question is, if you guys are planning for Firefox (or other browser support), when would it be available, if you guys have an approximate date?

**Comment by IronFlare** [comment]
Thank you for inquiring about the future of Trading Verifier!

Let me first apologize for the absurdly slow response. I hesitated to answer this issue for a long time in part to avoid adding fuel to a [Chrome vs. Firefox war](https://tenor.com/t16i.gif). To stop a protracted debate from occurring in the comments, I'll end this discussion with the following:

> Both browsers are [fairly](https://www.pcworld.com/article/3213031/computers/best-web-browsers.html?page=2) [comparable](https://www.businessinsider.com/google-chrome-vs-firefox-speed-performance-comparison-2018-7) when it comes to [performance](https://www.zdnet.com/article/whats-really-the-fastest-windows-10-web-browser-today/), Firefox has made great strides in recent years, and this is not the place to litigate the matter further.

To answer your question with a little bit of backstory, since the middle of last year I have been taking some much-needed time away from this extension to work on other hobbies and projects. In the meantime, the project has been in a quasi-LTS state, with support being offered for existing issues but no active development being done. Notably, however, at this time I am **not** ready to declare Trading Verifier end-of-life, especially given the many problems RLTracker's instability has caused me and my users in recent months.

Long story short, I wouldn't rule out the possibility of a Firefox version at some point in the future, but it would require a time investment that I'm not currently in a position to make for a number of personal and professional reasons that I won't get into.

If comments on this issue show there is significant demand for a Firefox version, I might seriously consider looking into it in the medium term.

Thanks,
IronFlare

**Comment by haltepunkt** [comment]
Any news on this?
